python/ray/util/horovod/BUILD

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------
+# Tests from the python/ray/util/horovod directory.
+# Please keep these sorted alphabetically.
+# --------------------------------------------------------------------
+
+py_test(
```

Comment:
do you have to add horovod_example.py too?

Reply:
It's being tested in `test_horovod` already